pkgs/applications/office/ticktick/default.nix

```diff
@@ -0,0 +1,73 @@
+{ lib
+, fetchurl
+, stdenv
+, wrapGAppsHook
+, dpkg
+, autoPatchelfHook
+, glibc
+, gcc-unwrapped
+, nss
+, libdrm
+, mesa
+, alsa-lib
+, xdg-utils
+, systemd
+}:
+stdenv.mkDerivation (final: {
+  pname = "ticktick";
+  version = "1.0.80";
+
+  src = fetchurl {
+    url = "https://d2atcrkye2ik4e.cloudfront.net/download/linux/linux_deb_x64/${final.pname}-${final.version}-amd64.deb";
+    sha256 = "1iyxn9rdv6p8d36swqpxb346arblk00yhv9z2qf6i6r2a4sbrbqh";
+  };
+
+  system = "x86_64-linux";
+
+  nativeBuildInputs = [
+    wrapGAppsHook
+    autoPatchelfHook
+    dpkg
+  ];
+
+  buildInputs = [
+    nss
+    glibc
+    libdrm
+    gcc-unwrapped
+    mesa
+    alsa-lib
+    xdg-utils
+  ];
+
+  # Needed to make the process get past zygote_linux fork()'ing
+  runtimeDependencies = [
+    systemd
+  ];
+
+  unpackPhase = ''
+    mkdir -p "$out/share" "$out/opt/${final.pname}" "$out/bin"
+    dpkg-deb --fsys-tarfile "$src" | tar --extract --directory="$out"
+  '';
+
+  installPhase = ''
+    cp -av $out/opt/TickTick/* $out/opt/${final.pname}
+    cp -av $out/usr/share/* $out/share
+    rm -rf $out/usr $out/opt/TickTick
+    ln -sf "$out/opt/${final.pname}/${final.pname}" "$out/bin/${final.pname}"
+    sed -i \
+      "s|Exec=/opt/TickTick/ticktick|Exec=$out/bin/${final.pname}|" \
+      "$out/share/applications/${final.pname}.desktop"
+  '';
+
+  meta = with lib; {
+    description = ''
+      TickTick is a powerful to-do & task management app with seamless cloud
+      synchronization across all your devices.
+    '';
```

Comment:
I would rather keep the description to one line and you dont need to use the name of it itself in the description. There is also `longDescription` if you want to have a place for extra text.